packages/backend/src/database/migrations/20240424131220_populate_slugs.ts

```diff
@@ -0,0 +1,45 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    const sanitizeSlug = (slug: string) =>
+        `REGEXP_REPLACE(
```

Comment:
unlike the regex in utils, this regex will not remove the trailing `-`  or combine multiple `--` , but to achieve that we will have to add nested regex... I think that's ok.